### ISSUE TITLE
Depth maps in training set and testing set

### ISSUE THREAD

**Issue by SPengLiang** [dingmyu/D4LCN#23]
Hello, I'm using the pre-trained model from Dorn, but I found that depth errors in KITTI testing set are much bigger than KITTI training set. I wanna know that is the issue that also existing in your experiments?  Thank you very much!

**Comment by dingmyu** [comment]
Hi SPeng,

Do you mean the error between the DORN depth map and the GT depth? Sorry we didn't measure this error. For both the training set and test set, we use the depth map as input and measure the 3D detection error.

Regards,
Mingyu

**Comment by SPengLiang** [comment]
Thank you for your reply. Firstly, I project the lidar to the image plane, and then as groundtruth to measure the depth error with predicted depth maps. However, I found that depth errors in the testing set are much higher than those in the training set.  Can you provide your depth maps in the testing set?  I'll very much appreciate it if you can. Thank you!

**Comment by dingmyu** [comment]
Sure, it can be found here.

> We also uploaded a copy in [Google Drive](https://drive.google.com/file/d/1JuDhHGH8DXzNkZSmaVrWyEhI3YuE2GqT/view?usp=sharing), Thanks.

_Originally posted by @dingmyu in https://github.com/dingmyu/D4LCN/issues/10#issuecomment-583955522_

**Comment by SPengLiang** [comment]
Okey! Thank you so much! You are really a so niceeeeeeee guy!

**Comment by dingmyu** [comment]
Thank you : )

Should you have further questions please feel free to reopen this issue.
